clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -69,6 +70,11 @@
     private final boolean autoCommitEnabled;
     private final AutoCommitTask autoCommitTask;
     private final ConsumerInterceptors interceptors;
+    private final boolean excludeInternalTopics;
+    
+    static {
+        INTERNAL_TOPICS.add("__consumer_offsets");
```

Comment:
@ijuma @granthenke Thank you for the feedback. What I understand is we should wait for KIP-4 to merge so we can leverage the boolean field here.